### ISSUE TITLE
[Tokenizer]: [A11y] Using Tokenizer for Deleting Filters - Clear All Button Is Not Accessible Via Keyboard

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#11282]
### Describe the bug

Actual Results

Clear all cannot be accessible via keyboard only operations

### Isolated Example

https://jvmiplyzwogithub-qyuo--5173--33edf5bb.local-credentialless.webcontainer.io/

### Reproduction steps

1.Go to https://jvmiplyzwogithub-qyuo--5173--33edf5bb.local-credentialless.webcontainer.io/
2. Using Tab key to get focus to the token item
3. The Clear All button could not be accessible
...

### Expected Behaviour

Expected Results 

Clear All button should be keyboard accessible

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP Successfactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @SAP/ui5-webcomponents-topic-rl ,

The reporter would like to be able to access the "clear all" Tokenizer button with keyboard navigation.

The issue is easily reproducible following this snippet:
https://stackblitz.com/edit/github-dr9gxkcb?file=src%2FApp.tsx

Could please take over?

Regards,
Plamen Ivanov

**Comment by hristop** [comment]
Hi @sleric1024 ,

The clear all does not need to be accessible via keyboard as the same result (or the functionality) can already be achieved via keyboard interaction -> Ctrl/Cmd + A and then Delete. The "Clear all" is more of a convenience for mobile users. 

Best Regards,
Hristo
